### ISSUE TITLE
[Designer Accessibility] Remove designer peer button should be keyboard accessible

### ISSUE THREAD

**Issue by anna-dingler** [microsoft/AdaptiveCards#7793]
# Issue

Keyboard accessibility needs to be enabled for the delete button on designer peers.

<img width="404" alt="image" src="https://user-images.githubusercontent.com/98650930/187734587-07e805b1-4acf-484a-9662-59d9c8d4bf0d.png">
